tests/jsonpath-addons/jsonpath-display-hints-prefixofsuffix.ref

```diff
@@ -0,0 +1,1221 @@
+{
+  "answerElements" : [
+    {
+      "class" : "org.batfish.question.jsonpath.JsonPathQuestionPlugin$JsonPathAnswerElement",
+      "results" : {
+        "0" : {
+          "displayValues" : {
+            "'nodes'->'as2border1'" : {
```

Comment:
I don't understand how this version of display values is useful. Given this and the information in the displayHints structure, what can I do with it?

I don't know the type of the key or the values, I don't have a text thing to put them in, etc.

Reply:
yeah, it is not complete from usability perspective. more tomorrow.